eslint/babel-eslint-parser/package.json

```diff
@@ -45,7 +42,7 @@
     "@babel/preset-flow": "^7.0.0",
     "@babel/preset-react": "^7.0.0",
     "dedent": "^0.7.0",
-    "eslint": "^6.0.1",
+    "eslint": "^6.0.0",
```

Comment:
Why this downgrade?

Reply:
I thought it was a good idea to match the `peerDep` with the `devDep`, and I think it makes sense to support the entire v6 release line. I can change it back if you prefer!

Reply:
I'll just leave it as is for now :)

Reply:
Well, the installed devdependency will be the same anyway